streamer/pulp/streamer/server.py

```diff
@@ -53,12 +55,16 @@ def __init__(self, request, streamer_config):
         :type  request:         twisted.web.server.Request
         :param streamer_config: The configuration for this streamer instance.
         :type  streamer_config: ConfigParser.SafeConfigParser
+        :param catalog_entry:   Catalog entry for the file requested.
+        :type  catalog_entry:   pulp.server.db.model.LazyCatalogEntry
+        :param pulp_request:    True if this request originated from Pulp.
```

Comment:
Can you elaborate on when or why a request would originate from outside of Pulp?